robottelo.properties.sample

```diff
@@ -232,6 +232,11 @@ ssh_key=
 # netmask=NETMASK_VALUE
 # bridge=BRIDGE_VALUE
 # gateway=GATEWAY_VALUE
+# dhcp_ipam=DHCP
```

Comment:
Can you please add all the validation strings details in the description of [vlan_networking] above, this way an individual would know what property to set and what not instead of wating for it to fail while running the test.

Reply:
done

Reply:
I was looking for updates to the docstring above vlan_networking. Because we still missing network/ipam. Anyways I will ACK.